records_mover/records/schema/field/pandas.py

```diff
@@ -70,7 +70,8 @@ def refine_field_from_series(field: 'RecordsSchemaField',
             if field.statistics is None:
                 field.statistics = statistics
             elif not isinstance(field.statistics, RecordsSchemaFieldStringStatistics):
-                raise SyntaxError("Did not expect to see existing statistics "
-                                  f"for string type: {field.statistics}")
+                raise ValueError("Did not expect to see existing statistics "
+                                 f"for string type: {field.statistics}")
```

Comment:
SyntaxError has some special rules around it; it's documented that the 'filename' field must be set.  As a result, when it's thrown out of code, the unit tests blow up when they try to read that field

ValueError works fine here, though.